manifests/03_credentials_request_aws.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: cloudcredential.openshift.io/v1
+kind: CredentialsRequest
```

Comment:
should this be called "ebs-credential-request"? In case we have to create separate credential request for EFS for example?

Reply:
Changed to `aws-ebs-csi-driver-operator` to align with the other CCs which are using the operator name.